src/test/java/com/wikia/webdriver/common/remote/Utils.java

```diff
@@ -26,25 +26,17 @@ public static String extractSiteIdFromMediaWikiUsing(final WebDriver driver) {
     return StringUtils.substringBetween(text, ": ", ",");
   }
 
-  private static String extractSiteIdFromMediaWiki(String wikiUrl) {
-    return new SiteId(wikiUrl).getSiteId();
-  }
-
   public static String excractSiteIdFromWikiName(String wikiName) {
-    String wikiUrl = UrlBuilder.createUrlBuilderForWiki(wikiName)
-        .getUrl()
-        .replace("https://", "http://");
-    return extractSiteIdFromMediaWiki(wikiUrl);
+    return new SiteId(wikiName).getSiteId();
   }
 
+  //TODO: Get rid of it and add services' url for fandom in config
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)